src/NSwag.ApiDescription.Client/NSwag.ApiDescription.Client.targets

```diff
@@ -0,0 +1,46 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project>
+  <PropertyGroup>
+    <_NSwagCommand>$(NSwagExe)</_NSwagCommand>
+    <_NSwagCommand Condition="'$(MSBuildRuntimeType)' == 'Core'">$(NSwagExe_Core21)</_NSwagCommand>
+  </PropertyGroup>
+
+  <!-- OpenApiReference support for C# -->
+
+  <Target Name="GenerateNSwagCSharp">
+    <ItemGroup>
+      <!-- @(CurrentOpenApiReference) item group will never contain more than one item. -->
+      <CurrentOpenApiReference>
+        <Command>$(_NSwagCommand) swagger2csclient /className:%(ClassName) /namespace:%(Namespace)</Command>
+      </CurrentOpenApiReference>
+      <CurrentOpenApiReference>
+        <Command>%(Command) /input:%(FullPath) /output:%(OutputPath) %(Options)</Command>
+      </CurrentOpenApiReference>
+      <CurrentOpenApiReference>
+        <Command Condition="! %(FirstForGenerator)">%(Command) /GenerateExceptionClasses:false</Command>
```

Comment:
Why do you set `GenerateExceptionClasses:false`?

Reply:
Because otherwise multiple C# generations in the same client project cause compiler errors. Note this is conditional and doesn't happen in the first (or only) C# generation in a project.